Client/src/app/_services/dark-mode.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { Observable, Subject } from "rxjs"
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DarkModeService {
+  private darkMode: boolean = true;
+  private subject = new Subject<any>();
```

Comment:
update the name to represent what it actually is. Also make use of type safety if possible instead of any